.travis.yml

```diff
@@ -58,6 +58,14 @@ jobs:
     - pip install -r tests/requirements.txt
     script:
     - make flake
+ - &MYPY
+    name: mypy
+    cache: false
+    python: "3.6"
```

Comment:
```suggestion
    python: "3.7"
```

Reply:
Makes total sense. Added 3.7 version for Travis-CI job.